.github/scripts/helpers.js

```diff
@@ -0,0 +1,15 @@
+exports.getRubyVersion = () => {
```

Comment:
With the reusable workflow, we don't need this file. These functions become Make targets, see [dotnet](https://github.com/Adyen/adyen-dotnet-api-library/blob/0d14bba85449d026a80e2cc71ef210a41d8efb5a/Makefile#L117) for examples.